schema/elements.go

```diff
@@ -74,6 +74,9 @@ type Struct struct {
 	// one or more fields in the above list. A given field from the above
 	// list may be referenced in exactly 0 or 1 places in the below list.
 	// Unions []Union `yaml:"unions,omitempty"`
+
+	// Atomic structs must not have individual items merged.
+	Atomic bool `yaml:"atomic,omitempty"`
```

Comment:
I think we should change these to be an enum to match fields before we use this?

Reply:
We don't know yet if it can be something else. I have read somewhere that we should avoid bools because they usually end-up being more than 2 states? :-)

Reply:
Yes, it's a code smell.

Reply:
I'll change in a follow-up.

Reply:
See #13